ui/app/AppMain.qml

```diff
@@ -26,6 +26,15 @@ StatusAppLayout {
     property alias appSettings: appSettings
     signal settingsLoaded()
 
+    property var currentPlayback: null
+
+    function stopPlayback() {
+        if (currentPlayback != null) {
+            currentPlayback.pause()
```

Comment:
Just one dumb question: why not `stop` method?

Reply:
there is no `stop` method, because there is no need to rewind to the beginning.